### PR TITLE
[INLONG-8365][Manager] InlongCluster support multiple tenant

### LINKED ISSUE #8365: [Feature][Manager] InlongCluster support multi-tenancy
### Description

parent #8275  #7914 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8365 

### Motivation

InlongCluster support multiple tenant

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml [hunk 1]
```diff
@@ -207,12 +210,14 @@
             ext_params           = #{extParams,jdbcType=LONGVARCHAR},
             description          = #{description,jdbcType=VARCHAR},
             heartbeat            = #{heartbeat,jdbcType=LONGVARCHAR},
+            tenant               = #{tenant,jdbcType=VARCHAR},
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml:213]: No need update this field.

File: inlong-manager/manager-web/sql/changes-1.8.0.sql [hunk 2]
```diff
@@ -94,3 +94,9 @@ ALTER TABLE `data_node`
     ADD `tenant` VARCHAR(256) DEFAULT 'public' NOT NULL comment 'Inlong tenant of datanode' after `description`;
 CREATE INDEX datanode_tenant_index
     ON data_node (`tenant`, `is_deleted`);
+
+
```
Comment (healchow) [inline inlong-manager/manager-web/sql/changes-1.8.0.sql:98]: One line is enough.

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml]: fixed, thx

File: inlong-manager/manager-web/sql/changes-1.8.0.sql (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-web/sql/changes-1.8.0.sql]: fixed, thx

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**